background/redux-slices/selectors/accountsSelectors.ts

```diff
@@ -381,7 +373,7 @@ function getNetworkAccountTotalsByCategory(
   return Object.entries(accounts.accountsData.evm[network.chainID] ?? {})
     .filter(([, accountData]) => typeof accountData !== "undefined")
     .map(([add, accountData]): AccountTotal => {
-      const address = getAddress(add) // TODO-MIGRATION temp fix
+      const address = add
```

Comment:
Better to rewrite `.map(([add, accountData])` => `.map(([address, accountData])`, so then you can remove `const address = add`.